inference/engine.py

```
@@ -9,26 +9,65 @@
 
 class InferenceEngine:
     def __init__(self, config, adapter_path: str):
```

Ruff (0.15.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)